Flepper.QueryBuilder/FlepperQueryBuilder.cs

```diff
@@ -23,12 +23,20 @@ public static ISelectCommand Select()
         public static ISelectCommand Select(params SqlColumn[] columns)
             => new QueryBuilder().SelectCommand(columns);
 
-        /// <summary>
-        /// Create Select Command
-        /// </summary>
-        /// <typeparam name="T">Object</typeparam>
-        /// <returns></returns>
-        public static ISelectCommand Select<T>() where T : class
+	    /// <summary>
+	    /// Create Select Command
+	    /// </summary>
+	    /// <param name="columns">Columns name</param>
+	    /// <returns></returns>
+	    public static ISelectCommand Select(string[] columns)
```

Comment:
Change `string[]`  for `params string[]`

Reply:
This change makes some of the tests fail to build:
`The call is ambiguous between the following methods or properties: 'FlepperQueryBuilder.Select(params SqlColumn[])' and 'FlepperQueryBuilder.Select(params string[])'
`